src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -324,7 +324,7 @@ public void writeBody(PrintWriter w) {
         @SuppressFBWarnings(value="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification="Jenkins.getInstance() is not null")
         public List<ResponseContributor> onNotifyCommit(String origin, URIish uri, String sha1, List<ParameterValue> buildParameters, String... branches) {
             if (LOGGER.isLoggable(Level.FINE)) {
-                LOGGER.fine("Received notification from " + StringUtils.defaultIfBlank(origin, "?")
+                LOGGER.fine("Received notification from " + StringUtils.defaultIfEmpty(origin, "?")
```

Comment:
Can you explain why you decided to switch to printed a string that is composed of only white space instead of printing the '?' character?  If origin is a string of white space (like ' '), the message will have changed from:

> Received notification from ? for uri = ...

to

> Received notification from  for uri = ...

That does not seem (to me) like it improved the message, but rather made it more difficult to understand.